azure-pipelines.yml

```diff
@@ -23,6 +23,8 @@ jobs:
   - script: |
       git clone -q --depth 1 git://github.com/Microsoft/vcpkg.git %VCPKG_ROOT%
       %VCPKG_ROOT%\bootstrap-vcpkg.bat
+    displayName: Bootstrap vcpkg
```

Comment:
This is what the `call` keyword is used for.

Reply:
Yes, you are right, fixed!